daemon/src/position_metrics.rs

```diff
@@ -343,6 +389,24 @@ mod metrics {
             .unwrap()
         });
 
+    static MARGIN_QUANTITY_GAUGE: conquer_once::Lazy<prometheus::IntGauge> =
+        conquer_once::Lazy::new(|| {
+            prometheus::register_int_gauge!(
+                "margin_quantities",
```

Comment:
🙃 I don't remember the naming conventions around metrics, but what about `margin` and `margin_counterparties` (or `margin_quantity` and `margin_counterparties_quantity` if the suffix is needed).

Reply:
Thanks for pointint this out. The convention is that the last word should indicate the type, e.e. in this case it should be `margin_satoshis`. 

I will fix this.

Reply:
Ah cool, that makes sense!